scripts/weather.coffee

```diff
@@ -19,16 +19,24 @@ module.exports = (robot) ->
         json = JSON.parse(body)
         lat = json.results[0].geometry.location.lat
         lng = json.results[0].geometry.location.lng
+        msg.send "Forecast for #{json.results[0].formatted_address}:"
       catch error
         msg.send "Error parsing location"
         msg.send error
-        lat = 0
-        lng = 0
+        return
       apikey = process.env.HUBOT_WEATHER_APIKEY
       forecasturl = "https://api.forecast.io/forecast/#{apikey}/#{lat},#{lng}"
       msg.http(forecasturl).get() (err, res, body) ->
         msg.send err if err
-        json = JSON.parse(body)
-        msg.send "#{json.currently.summary}, #{json.currently.temperature}°F, feels like #{json.currently.apparentTemperature}°F"
-      
-      
\ No newline at end of file
+        try
+          json = JSON.parse(body)
+          msg.send "Currently: #{json.currently.summary}, #{json.currently.temperature} °F, feels like #{json.currently.apparentTemperature} °F"
+          msg.send "Next hour: #{json.minutely.summary}"
+          msg.send "Today: #{json.daily.data[0].summary} H: #{json.daily.data[0].temperatureMax} °F L: #{json.daily.data[0].temperatureMin} °F"
+          msg.send "Tomorrow: #{json.daily.data[1].summary} H: #{json.daily.data[1].temperatureMax} °F L: #{json.daily.data[1].temperatureMin} °F"
+          if json.alerts
+            for alert in json.alerts
+              msg.send "#{alert.title} - #{alert.uri}"
+        catch error
+          msg.send "Failed to retrieve forecast."
+        
```

Comment:
Trailing whitespace! Also a missing newline, which is that the :no_entry_sign: here means :star: